server.js

```diff
@@ -141,7 +158,8 @@ function listen () {
 function initialise () {
   const app = unconfiguredApp
   initialiseSentry()
-  app.use(Sentry.Handlers.requestHandler()) // Sentry's request handler must be the first middleware on the app
+  configureSentryRequestHandler(app)
+  configureSentryErrorHandler(app)
```

Comment:
I believe the error handler should come after all of the routing/ server middleware and the request handler should come up front.

I'll see if I can find a reference for why I think that!

Reply:
I've just noticed that the error handler is in the middleware stack twice - would recommend replacing the bottom call with the new `confugreSentryErrorHandler` method

Reply:
For reference if anyone else reads this, from https://sentry.io/for/express/ : 
```
// The request handler must be the first middleware on the app
app.use(Sentry.Handlers.requestHandler());
// The error handler must be before any other error middleware
app.use(Sentry.Handlers.errorHandler());
```